From foo@baz Tue Apr  9 12:12:43 2002
To: Greg KH <greg@kroah.com>
Date: 02 Mar 2005 15:03:48 -08:00
From: khali@linux-fr.org
Subject: [PATCH] I2C: w83627hf needs i2c-isa

The w83627hf driver is useless unless i2c-isa is present. All other
drivers in this case do select I2C_ISA through Kconfig, so this one
should as well do.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff -Nru a/drivers/i2c/chips/Kconfig b/drivers/i2c/chips/Kconfig
--- a/drivers/i2c/chips/Kconfig	2005-03-02 16:31:44 -08:00
+++ b/drivers/i2c/chips/Kconfig	2005-03-02 16:31:44 -08:00
@@ -338,6 +338,7 @@
 	tristate "Winbond W83627HF, W83627THF, W83637HF, W83697HF"
 	depends on I2C && EXPERIMENTAL
 	select I2C_SENSOR
+	select I2C_ISA
 	help
 	  If you say yes here you get support for the Winbond W836X7 series
 	  of sensor chips: the W83627HF, W83627THF, W83637HF, and the W83697HF
